parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+2. Download and install the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/#installation) from the Blues Developer Experience site.
+3. Plug the Notecarrier containing the Notecard into your computer with a USB-A to micro USB cable.
+4. Copy the JSON file example in this folder named [`parking-lot-help-request-config-script.json`](./parking-lot-help-request-config-script.json) to your local machine.
+5. Remove all the # comments describing what each command does, replace the `product` and `sn` placeholders in the script with your Notehub Project UID (copied earlier when you created the project) and preferred Notecard serial number (e.g. `Lot G` or `G14` - depending on how location specific you want your help request buttons to be), and resave the file. It will resemble the code snippet below when you're done.
+```json
+{"req":"env.default","name":"parking_lot_help_request_config_script_version","text":"1.1.1"}
+{"req":"hub.set","product":"com.your-company.your-name:your_product", "sn":"preferred_device_name_here"}
+{"req":"hub.set","mode":"minimum","outbound":1440,"inbound":10080}
+{"req":"card.aux","mode":"neo-monitor"}
+{"req":"card.voltage","mode":"lipo"}
+{"req":"card.motion.mode","stop":true}
+```
+6. Upload the configuration script to the Notecard via the Notecard CLI by running the following command from the command line:
+```bash 
+$ notecard -setup parking-lot-help-request-config-script.json`
+```
+7. This will execute each of the JSON commands line by line against the Notecard and then it should be all set as far as firmware configuration goes.
+8. To confirm your Notecard is now attached to your project in Notehub, navigate to the project in the browser, click the **Devices** tab and see if your Notecard has appeared. If all goes according to plan, you'll see a device with the preferred name of your Notecard (provided by the setup script) under the **Best ID** column.
+  
+![Notecard registered to Notehub project with custom ID displayed as best ID instead of device's serial number](readme-notehub-device.png)
+_Devices tab on Notehub project showing Notecard's registered to this project._
+
+If you'd prefer to upload this script using the in-browser Notecard Playground instead of downloading the CLI, simply connect your Notecarrier to your computer via the USB-A to micro USB connector, pair it via serial port in your browser by clicking the **USB Notecard** button in the [Notecard Playground](https://dev.blues.io/notecard-playground/), and copy/paste the whole config file like in the screenshot below. (Don't forget to replace the placeholder `product` and `sn` variables in the script before copy/pasting.)
+
+![Pasting in the Notecard config script in the in-browser Notecard playground](./readme-notecard-playground-script.png)
+
+Now, click the button a few times and hopefully you'll see the LED light on the board flash and the button press data similar to this screenshot below flowing into your own Notehub project.
```

Comment:
As a suggestion, you might want to have the reader ensure that they’re still connected to the hardware from the other guide—because by the time you get to this point it’s been a minute since you’ve talked about the button at all.

That, or maybe a quick gif that shows pressing a button and seeing the lights 🙂 You might want a gif like that for Hackster anyways.